### PR TITLE
The move

### PR DESCRIPTION
**Pull request by t1gr0u**
#### What are the links to relevant tickets?
https://hxshortbreaks.atlassian.net/browse/WEB-4661

#### What does this PR do?
Move from bitbucket, added test coverage

#### What functional/unit tests does this PR have?
Started the unit test

#### How should a developer review this?
Install and test

#### Any background context you want to provide?
none

#### Screenshots (if appropriate)
screen shot 2015-04-21 at 16 24 00

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [ ] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1 @notABluesSinger
- [x] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2 @philmeehan
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

File: lib/dustjsExtended.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* jslint node: true */
+'use strict';
+
+/*
+* @name /lib/dustjsExtended.js
+* @description Extend dustjs-helpers with our own helpers.  This file bootstraps all functions in the `helpers` directory.
+* @since Sat Sep 13 2014
+* @author Kevin Hodges <kevin.hodges@holidayextras.com>
+*/
```
Comment (notABluesSinger) [inline lib/dustjsExtended.js]: all of the above block comment can come out

File: lib/templater.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/* jslint node: true */
```
Comment (notABluesSinger) [inline lib/templater.js]: This file name feel wrong, should it not be related to the repo?

Comment (philmeehan) [inline lib/templater.js]: The repo called templater?

File: lib/dustjsExtended.js (hunk 1 above)
Comment (philmeehan) [inline lib/dustjsExtended.js]: +1

**Comment by philmeehan** [comment]
I'm happy once the docblock that Rojo indicated has been removed.

File: lib/templater.js (hunk 2 above)
Comment (notABluesSinger) [inline lib/templater.js]: Perhaps it's just because the repo name has so many words in it.

**Pull request merged**